mapping/sustainability.yaml

```diff
@@ -1438,9 +1426,9 @@
     mapping: |-
         Project-level:
 
-        1. [Add a project document](../common.md#add-a-project-document), set `.documentType` to 'inclusiveDesign' and map any free text to `.description`.
+        1. [Add a project document](../common.md#add-a-project-document), set its [`.documentType`](project-schema.json,/definitions/Document,documentType) to 'inclusiveDesign' and map any free text to its [`.description`](project-schema.json,/definitions/Document,description).
```

Comment:
"Project-level" should be "Project level" above

Reply:
Fixed in 73bc7abfdb6a4bf562298f1f80ac2d6cfd7656ec